Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "Bijectors"
 uuid = "76274a88-744f-5084-9051-94815aaf08c4"
-version = "0.12.3"
+version = "0.12.4"
```

Comment:
Technically this should be a breaking release, but `Bijectors.stack` is used only in DPPL tests where it's qualified by module-name anyways.